Closet-Org/app/src/main/java/Organize/Closet/BottomItems.java

```diff
@@ -0,0 +1,81 @@
+package Organize.Closet;
+
+public class BottomItems implements Bottoms {
+  private String name;
+  private String color;
+  private String style;
+  private String occasion;
+  private String seasonalPurpose;
+
+  private String waistLine;
+  private boolean hasPockets;
+  private String bottomLength;
+
+  public BottomItems(
+      String name,
+      String color,
+      String style,
+      String occasion,
+      String seasonalPurpose,
+      String waistLine,
+      boolean hasPockets,
+      String bottomLength) {
+    this.name = name;
+    this.color = color;
+    this.style = style;
+    this.occasion = occasion;
+    this.seasonalPurpose = seasonalPurpose;
+    this.bottomLength = bottomLength;
+    this.waistLine = waistLine;
+    this.hasPockets = hasPockets;
+  }
+
+  @Override
+  public String getColor() {
+    return color;
+  }
+
+  @Override
+  public String getStyle() {
+    return style;
+  }
+
+  @Override
+  public String getName() {
+    return name;
+  }
+
+  @Override
+  public String getOccasion() {
+    return occassion;
+  }
+
+  @Override
+  public String seasonalPurpose() {
+    return seasonalPurpose;
+  }
+
+  @Override
+  public String bottomLength() {
+    return bottomLength;
+  }
+
+  @Override
+  public String getWaistLine() {
+    return waistLine;
+  }
+
+  @Override
+  public boolean hasPockets() {
+    return hasPockets;
+  }
+
+  public String describeItem() {
+    StringBuilder description = new StringBuilder();
+    description.append(String.format("Style: %s, Name: %s, Color: %s. ", style, name, color));
+    description.append(String.format("Occassion: %s, Seasonal: %s. ", occassion, seasonalPurpose));
```

Comment:
change to "occasion" on line 76 , then it should work 
`Occasion: %s, Seasonal: %s. ", occasion`